### PR TITLE
Add jupyterlab_nvdashboard

### LINKED ISSUE #143: Add back NVdashboard
### Context

We can now add [jupyterlab-nvdashboard](https://github.com/rapidsai/jupyterlab-nvdashboard) back to the base image, it now does support JLab 4

(I believe we removed the extension because it didn't suporrt JLab 4 at that time, ref: https://github.com/nebari-dev/nebari-docker-images/pull/101)

### Value and/or benefit

Useful for monitoring GPU resource utilization.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by pavithraes**
## Reference Issues or PRs

Closes #143

### CODE REVIEW

File: jupyterlab/environment.yaml [hunk 1]
```diff
@@ -58,6 +58,7 @@ dependencies:
   - pip:
       # vscode jupyterlab launcher
       - git+https://github.com/betatim/vscode-binder
+      - jupyterlab_nvdashboard==0.10.0
```
Comment (krassowski) [inline jupyterlab/environment.yaml:61]: this package is on conda-forge but the recipe appears unmaintained, the PR to bump to 0.10.0 is hanging here: https://github.com/conda-forge/jupyterlab-nvdashboard-feedstock/pull/17

Comment (pavithraes) [reply jupyterlab/environment.yaml:61]: Yeah, we can reach out to their team about that.

I think we can still install it through PyPI, which seems to be the latest version: https://pypi.org/project/jupyterlab-nvdashboard/

Comment (krassowski) [reply jupyterlab/environment.yaml:61]: @pavithraes it looks like a new release fixing the issue that was blocking us here is out, can we change the pin to [v0.11.0](https://github.com/rapidsai/jupyterlab-nvdashboard/releases/tag/v0.11.0) here?

```suggestion
      - jupyterlab_nvdashboard==0.11.0
```

Comment (marcelovilla) [reply jupyterlab/environment.yaml:61]: Is this something we want to incorporate in the next release of Nebari? Just trying to figure out what pending PRs we have to cut a final release this week.

Comment (pavithraes) [reply jupyterlab/environment.yaml]: Thanks! This should now be ready to merge :)

**Review by marcelovilla — APPROVED** [review]

**Pull request merged**